otcextensions/sdk/modelartsv1/v1/_proxy.py

```diff
@@ -9,9 +9,861 @@
+            delete a nonexistent service.
+
+        :returns: ``None``
+        """
+        return self._delete(
+            _service.Service, service, ignore_missing=ignore_missing
+        )
+
+    def get_service(self, service):
+        """Get details of a modelarts service.
+
+        :param service: service id or an instance of
+            :class:`~otcextensions.sdk.modelartsv1.v1.service.Service`
+
+        :returns: instance of
+            :class:`~otcextensions.sdk.modelartsv1.v1.service.Service`
+        """
+        return self._get(_service.Service, service)
+
+    def find_service(self, name_or_id, ignore_missing=False):
+        """Find a single service
+
+        :param name_or_id: The name or ID of a ModelArts service
+        :param bool ignore_missing: When set to ``False``
+            :class:`~openstack.exceptions.ResourceNotFound` will be raised
+            if the service does not exist.
+            When set to ``True``, no exception will be set when attempting
+            to find a nonexistent service.
+
+        :returns: instance of
+            :class:`~otcextensions.sdk.modelartsv1.v1.service.Service`
+        """
+        return self._find(
+            _service.Service, name_or_id, ignore_missing=ignore_missing
+        )
+
+    def update_service(self, service, **attrs):
+        """Update a Service Configurations.
+
+        :param service: service id or an instance of
+            :class:`~otcextensions.sdk.modelartsv1.v1.service.Service`
+        :param dict attrs: Keyword arguments which will be used to update
+            a :class:`~otcextensions.sdk.modelartsv1.v1.service.Service`,
+            comprised of the properties on the Service class.
+
+        :returns: instance of
+            :class:`~otcextensions.sdk.modelartsv1.v1.service.Service`
+        """
+        return self._update(_service.Service, service, **attrs)
+
+    def start_service(self, service):
+        """Start a Service.
+
+        :param service: service id or an instance of
+            :class:`~otcextensions.sdk.modelartsv1.v1.service.Service`
+
+        :returns: instance of
+            :class:`~otcextensions.sdk.modelartsv1.v1.service.Service`
+        """
+        return self._update(_service.Service, service, status="running")
+
+    def stop_service(self, service):
+        """Stop a Service.
+
+        :param service: service id or an instance of
+            :class:`~otcextensions.sdk.modelartsv1.v1.service.Service`
+
+        :returns: instance of
+            :class:`~otcextensions.sdk.modelartsv1.v1.service.Service`
+        """
+        return self._update(_service.Service, service, status="stopped")
+
+    def wait_for_service(self, service, timeout=1200, wait=5):
+        while timeout > 0:
+            obj = self.get_service(service)
+            status = obj.status.lower()
+            if status == "deploying":
+                pass
+            elif status in ["running", "finished"]:
+                return True
```

Comment:
why we need True here if we have no False option?

Reply:
method name is wait_for_service and its suppose to return on success otherwise it goes to Exception where it returns Timeout msg